builds/misc/templates/base-image-publish/base-image-overview.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
+- pwsh: |
+    # Note: With in the same job, you can access $(info), but it would interpret $(info) as string similar to `echo "$info"` which is undesirable for our case, we want an Object.
+    # We worked around this by using the serialization & deserailzation
+    $info = '$(serialize_info)' | ConvertFrom-Json
+
+    # Fetch tags & version from each module Dockerfiles {../$(arch)/base/Dockerfile, ../$(arch)/Dockerfile}
+    foreach ($each in $info)
+    {
+      $pathPrefix = "$(Build.SourcesDirectory)/$($each.Item1.ToString())/docker/$(os)/$(arch)"
+      $unberlyingFilePath = $pathPrefix + '/base/Dockerfile' | Resolve-path
+      $underlyingTag = $($(Get-Content $unberlyingFilePath | Select-String "ARG base_tag=") -split "=")[1]
+      $imageFilePath = $pathPrefix + '/Dockerfile' | Resolve-path
+      Set-Variable -Name "$($each.Item3.ToString())" -Value "$($($(Get-Content $imageFilePath | Select-String "ARG base_tag=") -split "=")[1])"
```

Comment:
Why do we set item3 here? Where do we use it later? It seems from the build script all we care about is the var name based on this with the "Version" suffix.

Reply:
Per the comment above, this is setting a variable name (value of item3) as a local var to be used in the next line 2 lines of code.